org.pitivi.Pitivi.json

```diff
@@ -0,0 +1,294 @@
+{
+    "app-id": "org.pitivi.Pitivi",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.22",
```

Comment:
I would strongly recommend 3.24 as it has had numerous improvements like supporting theming, VAAPI support, GStreamer 1.10 and other dependencies you use, etc.